examples/simple-game-server/fleet.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: "agones.dev/v1"
+kind: Fleet
+metadata:
+  name: simple-game-server
+spec:
+  replicas: 2
+  template:
+    spec:
+      ports:
+      - name: default
+        containerPort: 7654
+      template:
+        spec:
+          containers:
+          - name: simple-game-server
+            image: gcr.io/agones-images/simple-game-server:0.1
+            env:
+            # Disables the UDP listener (Enabled by default)
```

Comment:
Same here, feels like UDP should be enabled by default.

(Does it make sense to remove the env cars entirely for the default version, and maybe have a fleet-tcp.yaml for a TCP version?)

Reply:
Ah, that actually makes more sense. So remove the env vars from all the current yaml's, and just make a TCP-only fleet? That should show both how to disable UDP and enable TCP.